controls/radgridview/rows/selected-row-background.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Selected Row Background
+page_title: Selected Row Background
+description: Selected Row Background
+slug: radgridview-rows-selected-background
+tags: selected,row,background
+published: True
+position: 4
+---
+
+# Selected Row Background
+
+__RadGridView__ has a default __Brush__ for its selection on row level depending on the given theme. For example, the visual appearance of the selected row in the __Office2016__ theme would be as shown below.

```

Comment:
Do we need so much bolded text here? Can we remove the bolding from RadGridView? In my opinion it is clear that we are talking about it because we are in the RadGridView documentation.

Can we also remove the bolded Brush and Office2016 words?